### DISCUSSION TITLE
What do you understand about the Git merge conflict?

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#239] [Q&A] [answered]
Short answer.

**Comment by montasim** [comment, marked as answer]
A [Git merge conflict](https://www.simplilearn.com/tutorials/git-tutorial/merge-conflicts-in-git) is an event that occurs when Git is unable to resolve the differences in code between the two commits automatically. 

Git is capable of automatically merging the changes only if the commits are on different lines or branches.

![image](https://user-images.githubusercontent.com/95298623/197932515-6b93ff7b-c920-46ff-a256-164a485504ed.png)

**Comment by fcasamento** [comment]
Conflicts often arise when two people change the same lines in a file, or if one developer deleted a file while another was modifying it. Git makes this look easy to resolve by showing what the conflicts are.

**Reply by montasim** [reply]
@fcasamento  Try answering the questions that have no accepted answer. That way you will get unlock the next version of the Galaxy Brain badge.
